text_extraction_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notion
+
+import (
+	"encoding/json"
+	"testing"
+
+	notion "github.com/jomei/notionapi"
+	"github.com/matryer/is"
+)
+
+func TestExtractText_Paragraph(t *testing.T) {
+	is := is.New(t)
+
+	// Text in HTML: A paragraph with a link to <a href="https://conduit.io/">Conduit’s website</a>.
```

Comment:
is this added to make it easier for the reader to understand?

Reply:
Yup, because I think reading the JSON wouldn't help understand what the line actually is.